test/TrashToken.js

```diff
@@ -0,0 +1,185 @@
+const chai = require("chai");
+const chaiAsPromised = require("chai-as-promised");
+const { expect } = chai;
+chai.use(chaiAsPromised);
+const BN = require("bn.js");
+chai.use(require("chai-bn")(BN));
+const { expectEvent } = require("openzeppelin-test-helpers");
```

Comment:
How do you feel about putting some of this setup into a separate file?

Reply:
I'd be happy to see how much better it can get. I don't think much because we need some of those symbold available locally.

Reply:
Two options would be:
1. Store things like `expectEvent` on the global object; or
2. Export them from the setup file and destructure what we need in a single line from the test file. (Preferred)